semeio/jobs/config_jobs/PYSCAL

```diff
@@ -0,0 +1,23 @@
+STDERR PYSCAL.stderr
+STDOUT PYSCAL.stdout
+
+TARGET_FILE PYSCAL.OK
+
+PORTABLE_EXE ../scripts/fm_pyscal.py
+DEFAULT <RESULT_FILE> relperm.inc 
+DEFAULT <INT_PARAM_WO_NAME> __NOINTERPOLATION__
+DEFAULT <INT_PARAM_GO_NAME> __NOINTERPOLATION__
+DEFAULT <SHEET_NAME> 0
+DEFAULT <FAMILY> 1
+DEFAULT <SLGOF> SGOF
+ARGLIST <PARAMETER_FILE> <RESULT_FILE> <SHEET_NAME> <INT_PARAM_WO_NAME> <INT_PARAM_GO_NAME> <FAMILY> <SLGOF>
+
+MIN_ARG    1
+MAX_ARG    6
+ARG_TYPE 0 STRING
```

Comment:
The type checking does not really work and will not be fixed in its current format. Keep it as if if you like, but if it where me I would just remove it ;)